docs/usage.rst

```diff
@@ -100,80 +113,96 @@ Modifying Key and Parameter Group Assignments
 ---------------------------------------------
 
 Sometimes we see that there are important differences in acquisition parameters within a Key Group.
-If these differences impact how a pipeline will process the data, it makes sense to assign the scans
-in that Parameter Group to a different Key Group (i.e. assign them a different BIDS name). This can
-be accomplished by editing the empty columns in the `_summary.csv` file produced by ``cubids-group``.
+If these differences impact how a pipeline will process the data,
+it makes sense to assign the scans in that Parameter Group to a different Key Group
+(i.e., assign them a different BIDS name).
+This can be accomplished by editing the empty columns in the `_summary.csv` file produced by
+``cubids group``.
 
 Once the columns have been edited you can apply the changes to BIDS data using
 
 .. code-block:: console
 
-    $ cubids-apply /bids/dir keyparam_edited new_keyparam_prefix
+    $ cubids apply /bids/dir keyparam_edited new_keyparam_prefix
```

Comment:
this looks way cooler

Reply:
I just wish I could figure out a way to link from the subcommand code back to the sphinx-argparse-generated documentation, the way we can do `:func:'cubids.func'`